Providers/Scripts/2.4x-2.5x/Scripts/nxPackage.py

```diff
@@ -292,16 +293,16 @@ def __init__(self, Ensure, PackageManager, Name, FilePath, PackageGroup, Argumen
         self.cmds['dpkg'][
             'absent'] = 'DEBIAN_FRONTEND=noninteractive dpkg % -r '
         self.cmds['dpkg'][
-            'stat'] = "dpkg-query -W -f='${Description}|${Maintainer}|'Unknown'|${Installed-Size}|${Version}|${Status}|${Architecture}\n' "
+            'stat'] = "dpkg-query -W -f='${Description}#@#${Maintainer}#@#'Unknown'#@#${Installed-Size}#@#${Version}#@#${Status}#@#${Architecture}\n' "
```

Comment:
Is possible, could you create a delimeter variable at the top and then use string formatting -

```
>>> delimeter = "#@#"
>>> some_string = "dpkg-query -W -f ='${{Description}}{0}${{Maintainer}}{0}'Unknown'{0}${{Installed-Size}}{0}${{Version}}{0}${{Status}}{0}{{Architecture}}\n' ".format(delimeter)

# Output
"""
>>> print(some_string)
dpkg-query -W -f ='${Description}#@#${Maintainer}#@#'Unknown'#@#${Installed-Size}#@#${Version}#@#${Status}#@#{Architecture}
"""

```